### ISSUE TITLE
Change the direction of arrows when click on the navbar elements in simulator

### ISSUE THREAD

**Issue by Prerna-0202** [CircuitVerse/CircuitVerse#2627]
**Describe the bug**
When we click on the navbar elements the downward arrow remains in the downward position

**To Reproduce**
1. Go to 'circuitverse.org'
2. Go to 'Simulator'
2. Click on 'Project'

**Expected behavior**
When we click on the navbar elements and we hover on the dropdown elements the downward arrow  should change to an upward arrow

**Screenshots**
<img width="955" alt="2021-11-21 (10)" src="https://user-images.githubusercontent.com/89515816/142767486-501b2639-d9f1-4a80-b447-dd7cad75e750.png">

![jpg-change](https://user-images.githubusercontent.com/89515816/142767724-0fd3b509-098e-49d4-9a77-e896041a3b23.png)

**Device Information [optional]:**
 - OS: Windows
 - Browser :Chrome

**Additional context**

**Are you working on this issue? (Yes/No)**
Yes, I am working on this issue and will raise a pr soon.

**Comment by ajaybhaskar-h** [comment]
Can I work on this bug?

**Comment by Prerna-0202** [comment]
@ajaybhaskar-h, yes you can work on this issue...

**Comment by Hitansh-Shah** [comment]
I would also like to work on this issue. @ajaybhaskar-h Let me know if you are still working on this

**Comment by Palaksharma23** [comment]
i would like to work on this issue

**Comment by tichnas** [comment]
@Prerna-0202 Is this issue still available? If yes, I'd love to give it a shot!

**Comment by RahulSingh210** [comment]
If the bug is still present in the repo, I would like to work this out.

**Comment by Palaksharma23** [comment]
@RahulSingh210 currently, I am working on this issue 
try on different one please :).

**Comment by dipanshparmar** [comment]
Hey @Palaksharma23, are you still working on this?

**Comment by Palaksharma23** [comment]
you can take this up :) 👍

**Comment by dipanshparmar** [comment]
@Prerna-0202 Instead of changing the position of the arrows, why not just make them invisible as long as there is focus on that item because by default when we hover over the item the icon disappears. So let's just keep it invisible as long as there is the focus?

**Comment by Prerna-0202** [comment]
Yes, This can be done. Whatsay @tachyons?

**Comment by Snehagupta1907** [comment]
is anyone working on this bug?

**Comment by dipanshparmar** [comment]
Yeah @Snehagupta1907 I'm on it. I'm just waiting for the inputs from @tachyons

**Comment by tachyons** [comment]
cc @nitin10s

**Comment by rohit-raje-786** [comment]
@Prerna-0202 I have resolved the bug 🐞could just review it .

**Comment by aryanndwi123** [comment]
Closing the issue !!! As this is a stale Issue 🧮
